pkg/cloudprovider/providers/aws/aws_fakes.go

```diff
@@ -174,6 +174,10 @@ func (ec2i *FakeEC2Impl) DescribeInstances(request *ec2.DescribeInstancesInput)
 	return matches, nil
 }
 
+// DescribeInstancesPages is not implemented but is required for interface conformance
+func (ec2i *FakeEC2Impl) DescribeInstancesPages(input *ec2.DescribeInstancesInput, fn func(*ec2.DescribeInstancesOutput, bool) bool) error {
+	panic("Not implemented")
+}
```

Comment:
nit: new line between functions

```suggestion
}

```